app/models/concerns/eto/pathways_answers.rb

```diff
@@ -0,0 +1,228 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module Eto
+  module PathwaysAnswers
+    extend ActiveSupport::Concern
+    included do
+      def assessment_completed_on_answer
+        collected_at
+      end
+
+      def assessment_score_answer
+        # 2/17/2020
+        # Total boston pathways assessment score
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('scoring - only')
+        answer_from_section(relevant_section, 'pathways assessment score')&.to_i
+      end
+
+      def rrh_desired_answer
+        # 2/17/2020
+        # 9A
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('Why do people take RRH')
```

Comment:
Are these case sensitive? That it was all lowercase in `assessment_score_answer` has me confused...

Reply:
I standardized this and moved the down casing to HmisForm.  So, case insensitive now.